fleetscorebot/index.js

```diff
@@ -112,7 +112,9 @@ var getDevices = async function() {
   _.forEach(fleet, function(value, key) {
     var vers = key.split("%");
     var combo = { os: vers[0], supervisor: vers[1], count: value };
-    fleet_list.push(combo);
+    if (combo.os !== "Unknown") {
+      fleet_list.push(combo);
+    }
```

Comment:
Could do this maybe neater with a filter and map, but this is also fine for me.

Reply:
@CameronDiver redone the check as filter. Making the first step from foreach->map I'm currently unsure, as it's not an array but object that we receive. Didn't see a way to "map" on an object's keys, except foreach, which is the case here. But would be happy to learn :)

(that is `fleet` is an object, `fleet_list` is an array)

Reply:
I was thinking about lodash, which supports doing this kind of stuff with objects, but again I'm happy without the change.